crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -130,7 +130,9 @@ pub struct Diagnostic {
 impl Diagnostic {
     /// Add a new value as a [`DiagnosticMeasurement`].
     pub fn add_measurement(&mut self, measurement: DiagnosticMeasurement) {
-        if let Some(previous) = self.measurement() {
+        if measurement.value.is_nan() {
+            // Do nothing about the ema.
```

Comment:
I don't understand what you by "the ema" here.

Reply:
By the ema I mean self.ema. When the condition is hit none of the calculations for self.ema happen.